svgdigitizer/__main__.py

```diff
@@ -4,16 +4,24 @@
 def cli(): pass
 
 @click.command()
+@click.option('--sampling_interval',  help='specify sampling interval (for now in mV)')
```

Comment:
I think you should add a `sampling_interval` parameter to the `def` below. You can also specify here that this parameter is a `float` with the default `None`. Then you don't need any of the `if/else` logic below.

Reply:
Yes, is more elegant. Thanks.